docs/sample-v2-addon/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "sample-v2-addon",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Sample v2 addon for docs-app",
+  "keywords": [
+    "css",
+    "css-modules",
+    "ember-addon",
+    "emberjs",
+    "embroider",
+    "glint",
+    "typescript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ijlee2/embroider-css-modules.git"
+  },
+  "license": "MIT",
+  "author": "Isaac J. Lee",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": {
+      "types": "./dist/*.d.ts",
+      "default": "./dist/*.js"
+    },
+    "./addon-main.js": "./addon-main.cjs"
+  },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "dist/*"
+      ]
+    }
+  },
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "files": [
+    "addon-main.cjs",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:css": "stylelint \"**/*.css\" --cache",
+    "lint:css:fix": "stylelint \"**/*.css\" --fix",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "prelint:types": "type-css-modules --src src",
+    "lint:types": "glint",
+    "prepack": "rollup --config",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in tests/sample-v2-addon'"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.8.4"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.21.4",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
+    "@babel/preset-typescript": "^7.21.4",
+    "@babel/runtime": "^7.21.5",
```

Comment:
Here, I _had to_ add `@babel/runtime` to `sample-v2-addon` (a TS project). Otherwise, running `docs-app` (an Ember app that consumes `sample-v2-addon`) would result in an error.

<details>

<summary>Error message</summary>

```sh
[docs-app] ERROR in embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js 2:0-72
[docs-app] Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components'
[docs-app] resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components'
[docs-app]   Parsed request is a module
[docs-app]   using description file: embroider-css-modules/docs/sample-v2-addon/package.json (relative path: ./dist/components)
[docs-app]     Field 'browser' doesn't contain a valid alias configuration
[docs-app]     resolve as module
[docs-app]       embroider-css-modules/docs/sample-v2-addon/dist/components/node_modules doesn't exist or is not a directory
[docs-app]       embroider-css-modules/docs/sample-v2-addon/dist/node_modules doesn't exist or is not a directory
[docs-app]       looking for modules in embroider-css-modules/docs/sample-v2-addon/node_modules
[docs-app]         embroider-css-modules/docs/sample-v2-addon/node_modules/@babel/runtime doesn't exist
[docs-app]       embroider-css-modules/packages/node_modules doesn't exist or is not a directory
[docs-app]       looking for modules in embroider-css-modules/node_modules
[docs-app]         embroider-css-modules/node_modules/@babel/runtime doesn't exist
[docs-app]       node_modules doesn't exist or is not a directory
[docs-app]  @ ./components/hello-world.js 1:0-65 1:0-65
[docs-app]  @ ./templates/index.hbs 3:0-54 33:26-36
[docs-app]  @ ./assets/docs-app.js 62:13-46
[docs-app] 
[docs-app] webpack 5.81.0 compiled with 1 error in 4741 ms
[docs-app] Build Error (PackagerRunner) in embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js
[docs-app] 
[docs-app] Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
[docs-app] 
[docs-app] Stack Trace and Error Report: /var/folders/bg/<some hash>/T/error.dump.<some hash>.log
```

</details>

<details>

<summary><code>error.dump.log</code></summary>

```sh
  - broccoliBuilderErrorStack: ModuleNotFoundError: Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/Compilation.js:2016:28
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:817:13
    at eval (eval at create (embroider-css-modules/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (embroider-css-modules/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:448:22
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:118:11
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:689:25
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:893:8
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:1013:5
  - code: [undefined]
  - codeFrame: Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
  - errorMessage: embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js: Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
        at PackagerRunner (@embroider/webpack)
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js
    - line: [null]
    - treeDir: [undefined]
  - message: embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js: Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
        at PackagerRunner (@embroider/webpack)
  - name: Error
  - nodeAnnotation: @embroider/webpack
  - nodeName: PackagerRunner
  - originalErrorMessage: Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
  - stack: ModuleNotFoundError: Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in 'embroider-css-modules/docs/sample-v2-addon/dist/components/hello-world.js'
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/Compilation.js:2016:28
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:817:13
    at eval (eval at create (embroider-css-modules/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (embroider-css-modules/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:448:22
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:118:11
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:689:25
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:893:8
    at embroider-css-modules/node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:1013:5
```

</details>